src/Microsoft.PackageManagement.ArchiverProviders/project.json

```diff
@@ -3,7 +3,8 @@
 	"version": "1.0.0-*",
 
 	"buildOptions": {
-		"warningsAsErrors": true
+		"warningsAsErrors": true,
+                "debugType": "portable"
```

Comment:
again, the `debugType` setting should be moved to the `buildOptions` section within `netstandard1.6`, preferably right after the `"define"` property, because the `buildOptions` section here is shared by both `net451` and `netstandard1.6`. There are many changes in other project.json files having the same problem, please fix them all.